upload/docker/docker_test.go

```diff
@@ -36,7 +36,13 @@ func Test_parseRepositoryURI(t *testing.T) {
 			wantRepository: "simplesurance/calculator",
 			wantTag:        "latest",
 		},
-
```

Comment:
Having an empty server part and a repository name with multiple path components does not work.
The part before the first `/` is now misinterpreted as the server addr.

```
                {    
                        name:           "noServerLongRepo",                                                                                                                                                                                                    
                        destArg:        "simplesurance/app/calculator:latest",
                        wantServer:     "",
                        wantRepository: "simplesurance/app/calculator",
                        wantTag:        "latest",
                },
```